packages/frontend-2/lib/viewer/composables/serialization.ts

```diff
@@ -95,7 +96,11 @@ export function useStateSerialization() {
         filters: {
           isolatedObjectIds: state.ui.filters.isolatedObjectIds.value,
           hiddenObjectIds: state.ui.filters.hiddenObjectIds.value,
-          selectedObjectIds: [...state.ui.filters.selectedObjectIds.value.values()],
+          selectedObjectIds: selectedObjects.value.map((obj) => obj.id),
+          selectedObjectApplicationIds: selectedObjects.value.reduce((ret, obj) => {
```

Comment:
as long as selectedObjectApplicationIds always includes all selectedObjectIds as keys, it should be OK to drop the old selectedObjectIds key. you can always handle breaking changes across serialized state versions in that initializeMissingData function